README.md

```diff
@@ -120,68 +120,11 @@ ollama run mario
 Hello! It's your friend Mario.
 ```
 
-For more examples, see the [examples](examples) directory. For more information on working with a Modelfile, see the [Modelfile](docs/modelfile.md) documentation.
+For more examples, see the [examples](examples) directory. For more information on working with a Modelfile, see the [Modelfile](docs/modelfile.md) documentation. For details about using the CLI, refer to our [CLI Documentation](./docs/cli.md).
 
-## CLI Reference
+## Upgrading Ollama
 
-### Create a model
```

Comment:
Why was this deleted? We should still show the quick example commands here in the frontpage readme